README.md

```diff
@@ -13,6 +13,8 @@ joining.
 
 ## Using Import Tool
 
+Also see the [documentation for usage instructions](docs/usage.md).
```

Comment:
nit: starting a section with "Also ..." without saying anything before is a bit odd?

`Detailed documentation on the Import Tool is available [here](...)`

Reply:
done